extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -1,1101 +0,0 @@
-<?php
```

Comment:
Moved to process/ProcessBase.class.php, process/GlobalProcess.class.php and process/LocalProcess.class.php.